c-pallets/sminer/src/benchmarking.rs

```diff
@@ -130,12 +131,14 @@ benchmarks! {
         Sminer::<T>::timed_increase_rewards(RawOrigin::Root.into())?;
     }: _(RawOrigin::Root)
     verify {
-        assert!(<RewardClaimMap<T>>::contains_key(miner.clone())); 
+        assert!(<RewardClaimMap<T>>::contains_key(miner.clone()));
         let info = <RewardClaimMap<T>>::get(&miner).unwrap();
         let reward: BalanceOf<T> = 8000u128.try_into().map_err(|_e| "reward convert err")?;
         assert_eq!(info.total_reward, reward);
     }
 
+		 // should depend also on RewardClaimMap total items, there is a long
+		 // for loop with a some read/writes
```

Comment:
I have changed to rely on RewardClaimMaptotal items for calculation:
https://github.com/CESSProject/cess/pull/88/files#diff-443a91eceace517ab334a2fff82de9b0e7340c3e743a28508bf4e7bd8cad0ce4R147-R165